script/vsts/platforms/templates/preparation.yml

```diff
@@ -4,7 +4,11 @@ steps:
   - script: |
       sudo apt-get update
       sudo apt-get install -y wget software-properties-common
-      sudo apt-get install -y build-essential ca-certificates xvfb fakeroot git libsecret-1-dev rpm libx11-dev libxkbfile-dev xz-utils xorriso zsync libxss1 libgtk-3-0 libasound2 libicu-dev
+      sudo apt-get install -y build-essential ca-certificates xvfb fakeroot git libsecret-1-dev libx11-dev libxkbfile-dev xz-utils xorriso zsync libxss1 libgtk-3-0 libasound2 libicu-dev software-properties-common
```

Comment:
I notice that `software-properties-common` is on the higher `apt-get install` line already, frpm before the present PR.

(Not strictly related to this PR, but these two `apt-get install` lines could be combined for slightly faster execution, I think. And `--no-upgrade` would skip upgrading already-installed packages. Lastly, I wonder if the package cache is already populated, and if the first `apt-get update` can be skipped? That one takes a while to run. I can do these in a follow-up PR if you aren't interested in going admittedly a bit off-topic with these suggestions.)

Reply:
I will add these improvements, thanks

Reply:
Let's keep the first `apt-get update` in place though, to keep it fail-safe.
I don't think we should use `--no-upgrade`, because you actually might want to pull in a newer version of a package, like a security update, if it is available.